front/README.md

```diff
@@ -12,7 +12,7 @@ Then you should have the `elm` command available. To compile the application run
 
 <sub>*Please be sure to add `**/elm-stuff/` to your global git ignore file.</sub>
 
-### Get started
+## Get started
 
 You can serve up the frontend on it's own, or use spin the whole stack up together (see top level README)
```

Comment:
I'm not reading most of the unchanged parts, but this one stood out. `it's` should be `its`.